AccessManager.js

```diff
@@ -0,0 +1,143 @@
+/*!
+ * Copyright (c) 2019-2020 Digital Bazaar, Inc. All rights reserved.
+ */
+import {ProfileService} from 'bedrock-web-profile';
+
+export default class AccessManager {
+  /**
+   * Creates a new instance of an AccessManager. Should only be called
+   * by ProfileManager.
+   *
+   * @param {Object} options - The options to use.
+   * @param {Object} options.profile - The profile to manage access for.
+   * @param {string} options.profileManager - The parent `profileManager`
+   *  instance.
+   * @param {Object} options.users - A `users` Collection instance.
```

Comment:
Not sure *yet* what a users collection instance is, but maybe this should be `usersCollection`.

Reply:
I'm happy with the shorter name if you're amenable.